wordpress-mobile/WordPress-Android
# Fix changesConfirmed flag on page list

Corresponding [PR in FluxC](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1521)

This PR fixes a bug where changesConfirmedContentHashcode flag was getting lost after the app updated remotePostId when loading pages for page list. 

To test:
1. Turn on airplane mode
2. Create a new draft and enter a title
3. Click on Publish -> publish now
4. Notice, the label says "We'll publish the page, when your device is back online"
5. Turn off airplane mode and make sure the page gets published
----------------------
1. Turn on airplane mode
2. Create a new draft and enter a title
3. Press back
4. Notice, the label says "Local draft"
5. Turn off airplane mode and make sure the page gets uploaded as draft
----------------------
1. Turn on airplane mode
2. Open an existing published post
3. Edit its title
3. Press back
4. Notice, the label says "Local changes"
5. Turn off airplane mode and make sure the changes won't get published

PR submission checklist:
